src/pudl/transform/ferc1.py

```diff
@@ -4762,7 +4762,8 @@ def add_previous_year_factoid(self, df: pd.DataFrame) -> pd.DataFrame:
         ]
         # If previous_year type factoids aren't in all report_years, make factoids
         # for these years. Raise exception if more than one year.
```

Comment:
Is this comment still relevant? I"m not seeing the assertion it refers to anywhere.

Reply:
nope!